ipa-core/src/net/transport.rs

```diff
@@ -38,13 +38,13 @@ pub struct HttpTransport<F: ConnectionFlavor> {
 /// HTTP transport for helper to helper traffic.
 #[derive(Clone)]
 pub struct MpcHttpTransport {
-    inner_transport: Arc<HttpTransport<Helper>>,
+    pub(super) inner_transport: Arc<HttpTransport<Helper>>,
```

Comment:
this should be fine as long as we plan to get rid of the wrapper struct (which I believe is what we want) - seems that `HttpTransport<Helper>` gives us what we want from it

Reply:
In a follow up PR I add shard count to the shared http transport. Just FYI